master/setup.py

```diff
@@ -391,6 +391,10 @@ def define_plugin_entries(groups):
             'pylint==1.1.0',
             'pyflakes',
         ],
+        'bundle': [
+            'buildbot-www',
+            'buildbot-slave'
```

Comment:
its made for very early installs, in that case probably you want to run a slave on the master.
If we only depend on buildbot-www, another fix for that bug would be to make buildbot-www depend on buildbot, and ask people to install only buildbot-www if they need web, or buildbot if they dont.